src/entities/card-full/index.tsx

```diff
@@ -19,7 +19,12 @@ export const CardFull: FC<{ item: ICardContext }> = ({ item }) => {
       {!shopLogo && <Typography sx={titleStyle}>{shopName}</Typography>}
       {barcodeNumber && (
         <Box sx={{ ...barcodeStyle }}>
-          <Barcode displayValue={false} margin={0} value={barcodeNumber} />
+          <Barcode
+            displayValue={false}
+            margin={0}
+            value={barcodeNumber}
+            width={1.5}
```

Comment:
Супер! Я попробовал, на 1.5 у меня не влезает самый длинный баркод. Кажется лучше 1.3

```suggestion
            width={1.3}
```

Reply:
Ты прав